src/mjolnir/graphfilter.cc

```diff
@@ -27,10 +29,157 @@ uint32_t n_original_nodes = 0;
 uint32_t n_filtered_edges = 0;
 uint32_t n_filtered_nodes = 0;
 uint32_t can_aggregate = 0;
+uint32_t aggregated = 0;
 
 // Group wheelchair and pedestrian access together
 constexpr uint32_t kAllPedestrianAccess = (kPedestrianAccess | kWheelchairAccess);
 
+struct EdgeId {
+  uint64_t way_id;
+  GraphId graph_id;
+};
```

Comment:
seems this isn't used anywhere?

Reply:
Ah leftover from porting code from restriction builder.  ty